src/app/public/src/modules/table-of-contents/table-of-contents.component.scss

```diff
@@ -28,11 +31,11 @@
 }
 
 .stache-table-of-contents-wrapper {
-  border-left: 4px solid $stache-background-color-primary-dark;
-  padding: 6px 0 12px 19px;
+  padding: 6px 0;
 }
 
 .stache-table-of-contents-heading {
-  margin: 0 0 .7em;
-  font-size: 1.1em;
+  font-size: 1.5em;
+  font-weight: 300;
+  border-left: 8px solid transparent;
```

Comment:
Can we achieve the same affect by using padding or margin?  Seems strange to use a border transparent to adjust a block element.

Reply:
We use it in the Sidebar because the title can also be the `active` route, so it sometimes get filled with color.  Here it wouldn't make sense.

Reply:
If I remember correctly I was using this to assist in padding so when the blue line/border didn't exist it wasn't disappearing, it was just becoming transparent. This makes it so that the text is always a consistent distance away from the blue indicator.

Reply:
Correct, but in this case, we never set the `Content` header to have a border, so it won't shift.